.gitignore

```diff
@@ -11,3 +11,4 @@ dist/
 kustomize
 operator-sdk
 kubectl-directpv_*
+directpv.tar
```

Comment:
it was added to the `.gitignore` because it is generated during testing. 
https://github.com/minio/directpv/blob/master/docs/developer.md the image tar ball was created by podman

Reply:
Add a new line